Scripts/Interfaces/FunctionalityInjection/IControlSpatialHinting.cs

```diff
@@ -0,0 +1,113 @@
+﻿#if UNITY_EDITOR
+using UnityEditor.Experimental.EditorVR.Modules;
+using UnityEngine;
+
+namespace UnityEditor.Experimental.EditorVR
+{
+	/// <summary>
+	/// Gives decorated class ability to control spatial-hinting visuals
```

Comment:
and spatial-hinting visuals are _x_
This interface is the main place our users would be interacting with this, so we should be as verbose as neccessary here